### PR TITLE
Adding additional info field in dbCheckpoint status

### PR DESCRIPTION
**Pull request by nkumar04**
Added additional info field to show low disk space etc. in db checkpoint status command

### CODE REVIEW

File: reconfiguration/cmf/concord.cmf [hunk 1]
```diff
@@ -264,6 +264,7 @@ Msg GetDbCheckpointInfoRequest 59 {
 
 Msg GetDbCheckpointInfoStatusResponse 62 {
     list DbCheckpointInfo db_checkpoint_info
+    string additional_info
```
Comment (itsmeakashgoyal) [inline reconfiguration/cmf/concord.cmf:267]: nit: can we use a readable name in place of additional_info like available disk space or anything?

File: bftengine/src/bftengine/DbCheckpointManager.cpp [hunk 2]
```diff
@@ -377,4 +377,25 @@ BlockId DbCheckpointManager::getLastReachableBlock() const {
...
+    if (isSpaceBelowThreshold) {
+      ss << "Available disk space is less than threshold, free:" << HumanReadable{diskSpace.available}
+         << " rocksDbSize:" << HumanReadable{dbCurrentSize}
+         << " thresholdFactor:" << std::to_string(lowDiskThresholdFactor);
+    }
+  } catch (std::exception& e) {
+    ss << "Failed to get the available db size on disk: " << e.what();
+    LOG_ERROR(getLogger(), ss.str());
+  }
+  return ss.str();
```
Comment (itsmeakashgoyal) [inline bftengine/src/bftengine/DbCheckpointManager.cpp:399]: What this functional return if the space is not below threshold?

**Review by itsmeakashgoyal — APPROVED** [review]
Some minor comments else everything looks ok.
Thanks Nikhil

File: reconfiguration/cmf/concord.cmf (hunk 1 above)
Comment (nkumar04) [reply reconfiguration/cmf/concord.cmf:267]: kept name generic. "available disk space" is actually the information it gives

**Pull request merged**